### PR TITLE
SC-5858: a11y dropdowns

### PR DESCRIPTION
**Pull request by haniapalac**
# Description

Changes for all dropdowns except generated by feathers-filter.js library

## Links to Tickets or other pull requests

https://ticketsystem.hpi-schul-cloud.org/browse/SC-5858
integration test PR: https://github.com/hpi-schul-cloud/integration-tests/pull/192

## Changes

So far dropdowns were mainly rendered with the use of the chosen.js library which is not accessible (labels and options are not read out by screen readers which is a huge blocker)
it is a known issue: https://github.com/harvesthq/chosen/issues/264
The library has been removed and the dropdowns are rendered as native html elements.

## Screenshots of UI changes

e.g.
image
image
image
image
image
image

## Approval for review
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definition of Done
More and detailed information on the *definition of done* can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

**Comment by haniapalac** [comment]
Retry DeepCode

File: locales/de.json [hunk 1]
```diff
@@ -401,6 +401,7 @@
 				"student": "Schüler:in",
 				"studentParticipants": "Schüler / Teilnehmer",
 				"substituteTeacher": "Vertretungslehrer:in",
+				"noSubTeacherSelection": "Lehrer:in auswählen",
```
Comment (tofixx) [inline locales/de.json:404]: avoid double revocation
```suggestion
				"TeacherSelection": "Lehrer:in auswählen",
```

File: static/styles/lib/base.scss [hunk 2]
```diff
@@ -450,3 +450,7 @@ form.inline {
 	background-color: transparent;
 	border: none;
 }
+
+select {
```
Comment (tofixx) [inline static/styles/lib/base.scss:454]: really set all select width to 100%?

File: locales/de.json (hunk 1 above)
Comment (haniapalac) [reply locales/de.json:404]: @luizzaly do we need these placeholders? If not I will remove them altogether

File: static/styles/lib/base.scss (hunk 2 above)
Comment (haniapalac) [reply static/styles/lib/base.scss:454]: 100% width of the parent container. :-) so they are not too narrow when there are no options provided.

**Review by sarahschaefer — APPROVED** [review]

**Review by tofixx — APPROVED** [review]

**Pull request merged**